prbt_hardware_support/test/integrationtests/integrationtest_stop1.cpp

```diff
@@ -231,12 +234,13 @@ TEST_F(Stop1IntegrationTest, testServiceCallbacks)
   /**********
    * Step 3 *
    **********/
-  BARRIER("recover_callback");
+  BARRIER("unhold_callback");
   {
     InSequence dummy;
 
     // Call from Disconnect
     EXPECT_CALL(manipulator, holdCb(_,_)).Times(1);
+    EXPECT_CALL(manipulator, isExecutingCb(_,_)).Times(AtLeast(1));
```

Comment:
Well, because the service is called repeatedly. But I guess it was not designed for that and potentially leads to instabilities ...

After thinking it through I see the problem caused by the controller implementation. In my opinion the controller should either
- provide means for waiting until the hold position is reached
- wait until hold position is reached before returning the hold service. PR: #212

In order to make progress I will proactively use #212 to resolve the issues with is_executing in this context.